Marlin/src/pins/stm32f1/pins_FYSETC_AIO_II.h

```diff
@@ -113,6 +113,21 @@
 
 #endif
 
+#if HAS_TRINAMIC_CONFIG
```

Comment:
For some pins files these only apply with `HAS_TMC_UART`. However, the configurations define slave addresses any time that `HAS_TRINAMIC_CONFIG` is set. Which is better for these cases?

Reply:
The AIO II has more than one version of the board. One with 4988 and one with TMC drivers. I _thought_ they used 2208 drivers, but must be 2209 if they have addresses for them. If they do have 2208 versions the addresses _might_ interfere with communication. I don't know if the value is ignored, or if it is coerced to zero in that case.

Reply:
is the "A4988" board is in really ST820. but seems is pin compatible with A4988.

also, exist a 3rd board with TMC2209. but is not released yet